src/contribute/contribution-guide.md

```diff
@@ -383,6 +383,14 @@ At this point, you want to ensure everything is right. Test it with `mvn verify`
 
 Done. You can delete the local `finish-pr-<pull-request-#>` branch if you like.
 
+It's also possible to use the "Merge Pull Request" button on GitHub, supporting three modes:
+
+* Merge
+* Squash & Merge
+* Rebase & Merge
+
+It's similar to do the merge by hand.
```

Comment:
We should probably provide guidance on which one to use, or at least a pointer to the discussion thread.